### PR TITLE
[webui] Catch UnknownObject exception in repositories_controller

### PR DESCRIPTION
**Pull request by ChrisBr**
just c&p from require_package before filter in packages_controller.
Missed while introducing repositories_controller in #1968.

### CODE REVIEW

**Comment by hennevogel** [comment]
Uhm can you at least add a FIXME so we don't forget to DRY this...

**Comment by ChrisBr** [comment]
@hennevogel ok

**Comment by hennevogel** [comment]
Respect the Robot!

File: src/api/app/controllers/webui/repositories_controller.rb [hunk 1]
```diff
@@ -275,7 +275,14 @@ def calculate_repo_cycle(arch)
 
   def find_repository_parent
     if params[:package]
-      @main_object = @package = Package.get_by_project_and_name(@project.to_param, params[:package], use_source: false, follow_project_links: true)
+      # FIXME: Handle APIException different, this is just c&p from packages_controller
+      begin
+        @main_object = @package = Package.get_by_project_and_name(@project.to_param, params[:package], use_source: false, follow_project_links: true)
```
Comment (bgeuken) [inline src/api/app/controllers/webui/repositories_controller.rb]: Yikes. Double equal:(

**Pull request merged**